app/javascript/application.js

```diff
@@ -0,0 +1,35 @@
+// Entry point for the build script in your package.json
+import '@hotwired/turbo-rails';
+import './controllers';
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import { configureStore } from '@reduxjs/toolkit';
+import thunk from 'redux-thunk';
+import Greeting from './Greeting';
+import reducer from './reducer';
+
+const store = configureStore({
+  reducer,
+  middleware: (getDefaultMiddleware) => getDefaultMiddleware().prepend(thunk),
+});
+
+function App() {
+  return (
+    <Router>
+      <Routes>
+        <Route path="/" element={<Greeting />} />
+      </Routes>
+    </Router>
+  );
+}
```

Comment:
- [ ] Kindly make sure there is a component named `App`, you can move this code the the component instead